### PR TITLE
Resolve #1386, #1427, #1426

### LINKED ISSUE #1386: [Test] Receipt page
Verify info shown on the page matches mock transfers

Verify by using
-  transferId
   - logged in
   - not logged in
- receivingId
   - logged in
   - not logged in

Get receivingId by using getTransfer(transferId),
receivingId is not masked out for test env.

### PR DESCRIPTION
**Pull request by InVincible2016**

### CODE REVIEW

File: src/tests/e2e/receipt.test.e2e.js [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+  beforeAll(async () => {
+    await resetUserDefault()
+  }, timeout)
+
+  afterEach(async () => {
+    await jestPuppeteer.resetBrowser()
+  })
+
+  it(
+    `Transfer ID: ${transferDataList[0].transferId}`,
```
Comment (timothywangdev) [inline src/tests/e2e/receipt.test.e2e.js:28]: Should be transferState to be more descriptive

Comment (timothywangdev) [reply src/tests/e2e/receipt.test.e2e.js:28]: Also add transferType  (Sender/Receiver)

File: src/tests/e2e/receipt.test.e2e.js [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+      await explorerPage.close()
+
+      const { transferId } = await receiptPage.getReceiptFormInfo('transferId')
+      expect(transferId).toEqual(transfer.receivingId)
+    },
+    timeout
+  )
+
+  it(
+    `Receiving ID: ${transferDataList[2].receivingId} (SEND_DIRECT_TRANSFER_CONFIRMED) RECEIVER`,
```
Comment (timothywangdev) [inline src/tests/e2e/receipt.test.e2e.js:469]: do we have direct transfer in mock data?

File: src/tests/e2e/receipt.test.e2e.js [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+      await explorerPage.close()
+
+      const { transferId } = await receiptPage.getReceiptFormInfo('transferId')
+      expect(transferId).toEqual(transfer.transferId)
+    },
+    timeout
+  )
+
+  it(
+    `Transfer ID: ${transferDataList[2].transferId} (SEND_CONFIRMED_RECEIVE_CONFIRMED) SENDER`,
```
Comment (timothywangdev) [inline src/tests/e2e/receipt.test.e2e.js:200]: is it SEND_CONFIRMED_RECEIVE_PENDING?

File: src/tests/e2e/receipt.test.e2e.js (hunk 2 above)
Comment (InVincible2016) [reply src/tests/e2e/receipt.test.e2e.js:469]: typo. fixed

File: src/tests/e2e/receipt.test.e2e.js (hunk 3 above)
Comment (InVincible2016) [reply src/tests/e2e/receipt.test.e2e.js:200]: It is SEND_CONFIRMED_RECEIVE_CONFIRMED, "transfer completed"

**Pull request merged**